### PR TITLE
WEB: Blade Runner testing annoucement

### PR DESCRIPTION
**Pull request by peterkohaut**

### CODE REVIEW

File: data/news/20190616.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
+    "date": "1560679776",
+    "author": "petekohaut"
```
Comment (peterkohaut) [inline data/news/20190616.json]: i was able to misspell my own name, will fix on next update

**Comment by dafioram** [comment]
Any chance we can get the word retire in there? "Dangerous bugs will be retired"? Maybe a reference to voight-kampff?

**Comment by peterkohaut** [comment]
> 
> 
> Any chance we can get the word retire in there? "Dangerous bugs will be retired"? Maybe a reference to voight-kampff?

Good ideas! I've updated the text, but we don't have dangerous bugs anymore :)

**Comment by lotharsm** [comment]
Looks good to me!

One minor thing which is most like just my own opinion: I would change 
> unofficial Russian (Fargus Multimedia version)

to

> the unofficial Russian version by Fargus Multimedia.

**Comment by antoniou79** [comment]
Looks great to me! Good work!

**Pull request merged**